code/modules/hydroponics/beekeeping/bee_smoker.dm

```diff
@@ -0,0 +1,118 @@
+/// multiplier to decide how much fuel we add to a smoker
+#define WEED_WINE_MULTIPLIER 0.2
+/// how much using it costs
+#define SINGLE_USE_COST 5
+
+/obj/item/bee_smoker
```

Comment:
also turn `SINGLE_USE_COST` into a variable i want to set it to 0 and spray infinite bees as an admin to troll le players